src/jdk.jshell/share/classes/jdk/jshell/execution/RemoteExecutionControl.java

```diff
@@ -58,6 +58,8 @@ public class RemoteExecutionControl extends DirectExecutionControl implements Ex
      * @throws Exception any unexpected exception
      */
     public static void main(String[] args) throws Exception {
+        //disable System.console()
+        System.setProperty("jdk.console", "java.base");
```

Comment:
It would probably be more robust to have set on the debuggee command line but what you have is okay, as Console just reads the property rather using the "save properties" from startup.

Reply:
Thought of the same thing as Alan mentioned here, but I think it is OK too, as a temporary measure before JDK-8296454 is fixed.